### ISSUE TITLE
Update Readme

### ISSUE THREAD

**Issue by chashmeetsingh** [chashmeetsingh/Online-Quiz-Application#2]

**Comment by FleetAdmiral** [comment]
Will get right to it, could you mention any requirements you may have?

**Comment by chashmeetsingh** [comment]
Just explain the use of the app and how can the user install and run (complete procedure from cloning to running on browser)

**Comment by FleetAdmiral** [comment]
Alright, will get onto ASAP

**Comment by codegagan** [comment]
that should rather be into contribution.md file than in readme.md file

**Comment by FleetAdmiral** [comment]
@chashmeetsingh I'm still sticking to mentioning the use of the app and the installation/local deployment int he README seeing how the project isn't deployed (no mention of it anywhere), and so the only way for the user to use it is to locally deploy it? Additionally, the same would go into the contribution.MD (plus a little extra content if you'd like to add any), but fro now i'll finish the README.

**Comment by chashmeetsingh** [comment]
Yes, you can can add the steps for any user to deploy it on their system + the use of the app and the features which are allowed. @FleetAdmiral

**Comment by FleetAdmiral** [comment]
@chashmeetsingh A few issues regarding README. Firstly, i don't think the README should be crowded (since its the first doc a person views when visiting the project), so it would be advisable to skip the use/features of the app and maybe put that in another doc. 

Also, could you mention the way you locally deploy the project personally?

**Comment by chashmeetsingh** [comment]
@FleetAdmiral README is mean to explain the purpose of the project and also includes the steps one should take to install them on their local. So i would recommend to add them both. For the case of deploying instructions, I can't because I have discontinued the project.
Thanks!
